### PR TITLE
Milestone 1. To-Do-List Project list-structure branch

### PR DESCRIPTION
**Pull request by otmanTR**
In this branch following changes done;
-added linters
-added Webpack file
-added index.html file
-added index.js file
-added style.css file
-fixed linters
-updated readme file
-MIT licance updated

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  },
+  {
+    discription: 'Create a styler t List',
+    completed: false,
+    index: 4,
+  },
+];
+
+const taskList = document.querySelector('.taskList');
+// eslint-disable-next-line no-unused-vars
```
Comment (mckent05) [inline src/index.js]: - Please remove all disabled linters from your code base, remember linters help to ensure that we write code that is clean and easier to maintain. Please fix this 😄

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  },
+  {
+    discription: 'On page  tasks ',
+    completed: false,
+    index: 2,
+  },
+  {
+    discription: 'The list should aalues for each task.',
+    completed: false,
+    index: 3,
```
Comment (mckent05) [inline src/index.js]: - Please ensure that all objects in your array have the same keys i.e `description, completed, index`, there are some differences due to spelling. Please fix this 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @otmanTR, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// import _ from 'lodash';
+import './style.css';
+
+const tasks = [
+  {
+    description: 'Write a  ',
+    complated: false,
```
Comment (fobadara) [inline src/index.js:7]: [OPTIONAL] Kindly change this to completed with `e`

**Review by fobadara — APPROVED** [review]
Hi @otmanTR, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**